chain/sync.go

```diff
@@ -528,8 +528,16 @@ func blockSanityChecks(h *types.BlockHeader) error {
 
 // Should match up with 'Semantical Validation' in validation.md in the spec
 func (syncer *Syncer) ValidateBlock(ctx context.Context, b *types.FullBlock) error {
+	validationStart := time.Now()
+	defer func() {
+		durationMillisec := float64(time.Since(validationStart).Round(time.Microsecond).Nanoseconds()) / float64(time.Millisecond)
```

Comment:
@Kubuxu Maybe slightly less expensive to do `time.Since(...).Seconds() * float64(1000)`?

Reply:
Should be roughly equivalent but yeah, good idea.